### PR TITLE
Flipping function

### LINKED ISSUE #11: Write function specifications and documentation - Flipping

### PR DESCRIPTION
**Pull request by Marcony1**
Added py file with Flipping function docstring.

### CODE REVIEW

**Review by SoloSynth1 — DISMISSED** [review]
LGTM - I was thinking about adding [type hinting](https://docs.python.org/3/library/typing.html) but that's entirely optional.

**Review by SoloSynth1 — COMMENTED** [review]
On second thought would it be better if we output a numpy array instead of a dataframe?

**Comment by Marcony1** [comment]
Thank you for your comments @SoloSynth1 Output was changed in the docstring

File: src/visumorph/flipping.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    Parameters
+    ----------
+    input_file : str
+        Path to image file.
+    v : int, optional
+        Flip vertically if v=1, horizontally if v=0. By default, v=0.
+
+    Returns
+    -------
+    df
```
Comment (SoloSynth1) [inline src/visumorph/flipping.py]: I think we need to specify of type of returned value here as specified by the [numpy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#returns)

As name for returned value is optional, I would suggest we don't specify it for this moment for greater flexibility. Please also note that `df` is usually associated with variable of type `pandas.DataFrame` by convention.
```suggestion
    ndarray
```

**Review by SoloSynth1 — CHANGES_REQUESTED** [review]

**Comment by Marcony1** [comment]
Forgot to change that one. Thank you!

**Review by SoloSynth1 — APPROVED** [review]
LGTM, merging to `main`. Thanks @Marcony1

**Pull request merged**